### DISCUSSION TITLE
Typescript eslint help

### DISCUSSION THREAD

**Discussion by NaviTheCoderboi** [typescript-eslint/typescript-eslint#9208] [Technical Discussions]
### Body

guys pls help me to fix this fk error,i am fade up from it
```ts
Parsing error: ESLint was configured to run on `<tsconfigRootDir>/eslint.config.js` using `parserOptions.project`: <tsconfigRootDir>/tsconfig.json
However, that TSConfig does not include this file. Either:
- Change ESLint's list of included files to not include this file
- Change that TSConfig to include this file
- Create a new TSConfig that includes this file and include it in your parserOptions.project
See the typescript-eslint docs for more info: https://typescript-eslint.io/linting/troubleshooting#i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-fileeslint
```
i getting this even i am ignoring the file
```ts
import base from '@repo/eslint-config/base.js';

export default [
    ...base,
    {
        files: ['src/**/*.{js,ts}'],
        ignores: ['eslint.config.js', 'test']
    }
]
```

i am using typescript eslint

### Additional Info

Any additional info...

### Before you submit your discussion, please confirm the following. If any of these required steps are not taken, we may not be able to review your RFC. Help us to help you!

- [X] I have [read the discussions guidelines](https://typescript-eslint.io/contributing/discussions) and per those guidelines, this fits a discussion category, _not_ a help request or standard issue.
- [X] I have [searched for related discussions](https://github.com/typescript-eslint/typescript-eslint/discussions) and [searched for related issues](https://github.com/typescript-eslint/typescript-eslint/issues) and found none that match my proposal.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

**Comment by auvred** [comment]
> * I have [read the discussions guidelines](https://typescript-eslint.io/contributing/discussions) and per those guidelines, this fits a discussion category, _not_ a help request or standard issue.

As per our discussion guidelines we don't use discussions for support. 

---

> * I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

Please see the FAQ: https://typescript-eslint.io/troubleshooting/#i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file

**Comment by bradzacher** [comment]
> guys pls help me to fix this fk error,i am fade up from it

Also please watch your tone and language and follow the code of conduct.

https://github.com/typescript-eslint/typescript-eslint/blob/main/CODE_OF_CONDUCT.md
